README.md

```diff
@@ -55,7 +55,12 @@ pre-commit install
 
 ### Run unit tests
 ```
-pytest
+tox
+```
+
```

Comment:
Maybe could be expanded something like this? (if this is still accurate)

```suggestion
Tests can be run with either `tox` or `pytest`.  

To run all the tests in a single test file, use pytest and specify the path to the test: `pytest tests/test_dateformat/test_base.py`

To test cases by name, use pytest: `pytest -k test_str`

Reply:
Thats a good idea to update this more. The idea of the readme is, that its content is reused in the readthedocs documentation, and thus updating content in one place is enough.

Reply:
yes, agreed — like the idea of using the readme for both places